readme.md

```diff
@@ -12,13 +12,30 @@ To enable the processor please add the following rule to the `transform` object
 "jest": {
     // ...
     transform: {
+        "\\.js$": "babel-jest"
         "\\.hbs$": "jest-handlebars",
     }
     // ...
 }
 ```
 
-Now all imported handlebars files will get compiled by the processor and the render function is imported.
+Now all imported handlebars files will get compiled by the processor and the render function is imported. You will need to add `babel-jest` for `.js` files if you previously had no transforms added as the default.
+
+## Tests
+
+```hbs
+<!-- greetings.hbs -->
+<h1>Hello {{name}}</h1>
+```
+
+```js
+import greetingsHbs from './greetings.hbs';
```

Comment:
This code sample is an extended use case, the standard use case would be to allow your modules under test to use compiled handlebars templates. Maybe you can add this information in your PR?